testing/tests/ui.rs

```diff
@@ -1,9 +1,11 @@
+#![cfg(not(windows))]
```

Comment:
If we're going to rely on `std::os::unix`, should we use `#![cfg(unix)]` instead?